t/cobrand/brent.t

```diff
@@ -1243,9 +1341,16 @@ FixMyStreet::override_config {
                 return;
             }
             $mech->submit_form_ok({ with_fields => { name => "Test McTest", email => $user1->email } });
-            $mech->submit_form_ok({ with_fields => { 'process' => 'summary' } });
-            $mech->content_contains('Your container request has been sent');
+            if ($referral) {
+                $mech->submit_form_ok({ with_fields => { 'process' => 'summary' } });
+                $mech->content_contains('Your container request has been sent');
+            } else {
+                $mech->waste_submit_check({ with_fields => { 'process' => 'summary' } });
```

Comment:
This checks it has redirected to payment, right?

Reply:
Yes, using a clone of the mech to prevent host/cookie issues.